velox/dwio/parquet/writer/Writer.cpp

```diff
@@ -217,7 +218,20 @@ dwio::common::StripeProgress getStripeProgress(
  *
  * This method assumes each input `ColumnarBatch` have same schema.
  */
-void Writer::write(const VectorPtr& data) {
+void Writer::write(const VectorPtr& dataToWrite) {
+  VectorPtr data = dataToWrite;
+  if (data && schema_) {
+    auto rowType = std::dynamic_pointer_cast<const velox::RowType>(schema_);
+    auto rowVector =
+        std::dynamic_pointer_cast<facebook::velox::RowVector>(data);
+    data = std::make_shared<facebook::velox::RowVector>(
+        rowVector->pool(),
+        rowType,
+        rowVector->nulls(),
```

Comment:
This must be nullptr, no?

Reply:
@mbasmanova In this case, we only need to update the `rowType`. I am unsure why `nullptr `is being used instead of rowVector->nulls().

Reply:
Since this RowVector represents a table (not a struct column), it cannot have nulls. You may add VELOX_CHECK_NULL(rowVector->nulls()), then use nullptr.

Reply:
@mbasmanova  Updated.